report/os_xdg.go

```diff
@@ -1,3 +1,4 @@
+// +build linux openbsd freebsd netbsd dragonfly
```

Comment:
This is ineffective as there is no newline after

Reply:
Good catch @andydotxyz, fixed in aee5dbc
/cc @Jacalz

Reply:
Ah, nice catch. Thanks @andydotxyz. Will get my PR updated when I get a chance.